src/test/java/org/folio/rest/api/UpdateItemStatusDateFunctionMigrationTest.java

```diff
@@ -0,0 +1,108 @@
+package org.folio.rest.api;
+
+import static io.vertx.core.json.JsonObject.mapFrom;
+import static org.folio.rest.api.StorageTestSuite.TENANT_ID;
+import static org.folio.rest.support.matchers.DateTimeMatchers.withinSecondsBeforeNowAsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.startsWith;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.joda.time.Seconds.seconds;
+
+import java.net.MalformedURLException;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.folio.rest.jaxrs.model.TenantAttributes;
+import org.folio.rest.support.IndividualResource;
+import org.folio.rest.support.builders.ItemRequestBuilder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import io.vertx.core.json.JsonObject;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+
+@RunWith(JUnitParamsRunner.class)
+public class UpdateItemStatusDateFunctionMigrationTest extends MigrationTestBase {
+  @Parameters({
+    "18.2.3, 19.1.1",
```

Comment:
How does this case work when the `fromVersion` in the `schema.json` states `19.2.0`?

Reply:
That's confusing me too. It seems that a script won't be executed only when it's `fromModuleVersion` is less then `fromVersion`, otherwise it will be, even if it belongs to a 'future' release.

@julianladisch, am I correct understand?

Reply:
As agreed, I've added explanation for that case.